packages/calypso-e2e/src/lib/components/support-component.ts

```diff
@@ -156,16 +100,14 @@ export class SupportComponent {
 	 * Fills the support popover search input and waits for the query to complete.
 	 *
 	 * @param {string} text Search keyword to be entered into the search field.
-	 * @returns {Promise<void>} No return value.
 	 */
 	async search( text: string ): Promise< void > {
-		// Wait for the response to the request and ensure the status is HTTP 200.
+		await this.helpCenter.getByPlaceholder( 'Search for help' ).fill( text );
+
 		await Promise.all( [
-			this.page.waitForResponse( selectors.successfulSearchResponse ),
-			this.page.fill( selectors.searchInput, text ),
-			this.waitForQueryComplete(),
+			this.page.waitForResponse( /wpcom\?query/ ),
```

Comment:
I think adding a comment with a full example URL that we're matching here would be useful.

Reply:
Sounds good, will do. 

Comment and example added in 23ed24e76e93e4ec1b0598cbedc0e6ad2f89f4dd.